File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n=nums.size(), k=1;
        if(n==0) return 0;
        for(int i=0;i<n;i++){
            if(nums[i]!=nums[k-1]){
                nums[k]=nums[i];
                k++;
            }
        }
        return k;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n=nums.length;
        if(n==1) return 1;
        int[] arr=new int[n];
        arr[0]=nums[0];
        int i=1,j=1;
        while(j<n){
            if(nums[j-1]!=nums[j]){
                arr[i++]=nums[j];
            }
            j++;
        }
        for(int k=0;k<i;k++){
            nums[k]=arr[k];
        }
        return i;
    }
}